app.js

```diff
@@ -146,10 +155,10 @@ app.use(async (req, res, next) => {
 	res.locals.notification = req.session.notification;
 	res.locals.inline = req.query.inline || false;
 	setTheme(res);
-	res.locals.domain = process.env.SC_DOMAIN || 'localhost';
+	res.locals.domain = SC_DOMAIN;
 	res.locals.production = req.app.get('env') === 'production';
-	res.locals.env = req.app.get('env') || false;
-	res.locals.SENTRY_DSN = process.env.SENTRY_DSN || false;
+	res.locals.env = req.app.get('env') || false; // TODO: ist das false hier nicht quatsch?
```

Comment:
jap, daher NODE_ENV verwenden und in den globals process.env.NODE_ENV || 'development' setzen?
--> https://stackoverflow.com/a/34228447

Reply:
req.app.get('env') || false vs process.env.NODE_ENV || 'development' ist nicht unbedingt das gleiche. Bis auf den stackoverflow post habe ich noch nichts zu default values bei get('env') gelesen. 
Zum nachlesen fehlt mir die Zeit und ohne nachlesen ist es zu kritisch zum ändern.